scripts/docker_cmake

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+rm -f  CMakeCache.txt
+rm -rf CMakeFiles/
```

Comment:
A few comments here.  In our Jenkins CI build, we clear entirely the build directory (see the script [docker/jenkins/build.sh](https://github.com/ORNL-CEES/DataTransferKit/blob/master/docker/jenkins/build.sh#L7)), so that change would not affect the automated builds.  If we remove these guys we will have to recompile everything every time we reconfigure.